Align content within in the tables

garrett opened the pull request:
I think this closes #37?

Fixes #31

garrett commented:
Aha, I noticed an issue with the alignment of disks, as the string row is above the progress stack, instead of the other way around. I have an idea for an easy fix.

martinpitt commented:
By and large this is great, and the CPU/memory tables are now aligned with and without swap.

I just noticed two things:

![image](https://user-images.githubusercontent.com/200109/93219946-34a55300-f76c-11ea-878a-54022db9c693.png)

 * The "disks" mount points now get pushed to the bottom. At first sight I thought "uh, that's broken". Maybe it's intended, maybe it's not actually so bad. Should we restrict the flexing to the CPU/memory tables for now? Is that what you meant with your "alignment of disks" comment above?

 * The "network" table is misaligned with the other card contents. But this isn't actually a regression, current master already does that.

martinpitt commented on src/app.scss:
```diff
@@ -19,11 +19,30 @@
+      }
+
+      // Give the description list as much leftover space as possible,
+      // when it exists with other flexing elements
```
Let's add a comment like "This will push the cgroups tables to the bottom, so that they become aligned".

garrett replied:
Adjusted the comments to make this clearer.

garrett commented:
Changes between the original and what's here are in 60c13a8. I have since force-pushed.

garrett commented:
>    The "disks" mount points now get pushed to the bottom. At first sight I thought "uh, that's broken". Maybe it's intended, maybe it's not actually so bad. Should we restrict the flexing to the CPU/memory tables for now? Is that what you meant with your "alignment of disks" comment above?

Fixed. Yep. :wink:

>    The "network" table is misaligned with the other card contents. But this isn't actually a regression, current master already does that.

Due to `<th>`. I fixed this in the PR.

martinpitt commented:
Perfect, thank you! I just did a force-push to drop an excessive trailing space (that vim highlighted in bright red).
